### DISCUSSION TITLE
Google Social login failing

### DISCUSSION THREAD

**Discussion by lananeads** [cypress-io/cypress#25830] [Questions and Help]
Wondering if anyone ever seen this error and can Cypress really not handle google social login --

"`cy.task('GoogleSocialLogin')` failed with the following error:  > waiting for selector "a[href="/Account/Login"]" failed: timeout 30000ms exceeded  https://on.cypress.io/api/task
Stacktrace
CypressError: `cy.task('GoogleSocialLogin')` failed with the following error:"

any ideas? thank you!

**Comment by x-ghaith** [comment]
I am seeing it right now

**Reply by lananeads** [reply]
Hi @x-ghaith 
after reading through cypress documentation it became clear that Google login is now more difficult to login using automation testing tools like Cypress as they treat this as bot login so for security reasons Google will throw a capture each time which you cannot know in dance to script it. Therefore we are using a "backdoor" login to login into our systems, a token and does not expire for a very long time and you can reuse it over and over in your tests.

I hope this helps.
